arangod/Aql/AqlCall.cpp

```diff
@@ -60,32 +63,31 @@ auto AqlCall::fromVelocyPack(velocypack::Slice slice) -> ResultT<AqlCall> {
 
   auto const readLimit =
       [](velocypack::Slice slice) -> ResultT<AqlCall::Limit> {
-    auto const type = slice.type();
+    auto type = slice.type();
```

Comment:
Do we need this `type`? Why not just `slice.isString()` on the next line and remove this variable? As all way down we anyway use `slice.isXXXXX`

Reply:
Addressed